addons/website/static/lib/multirange/multirange_custom.js

```diff
@@ -295,11 +295,11 @@ class Multirange {
     }
 
     formatNumber(number) {
-        number = String(number).split('.');
-        if (number[1] && number[1].length === 1) {
-            number[1] += '0';
-        }
-        let formatedNumber = number[0].replace(/(?=(?:\d{3})+$)(?!\b)/g, ',') + (number[1] ? '.' + number[1] : '.00');
+        const locale = document.querySelector("html").getAttribute("lang");
+        let formatedNumber = number.toLocaleString(locale, {
+            minimumFractionDigits: 2,
+            maximumFractionDigits: 2,
+        });
```

Comment:
If you were to refactor the multirange input, it would be great to provide a standard currency code for the `this.currency` variable. It would allow us to do something like this:
```js
const formattedNumber = number.toLocaleString(locale, { style: "currency", currency: this.currency });
```
which will take care of the positioning of the currency symbol for us and fix the point to two fractional digits